ui/src/scenes/Home/components/NeedsContainer/NeedsContainer.js

```diff
@@ -4,7 +4,6 @@ import gql from 'graphql-tag';
 import _ from 'lodash';
 import { withRouter, Redirect } from 'react-router-dom';
 import { Query } from 'react-apollo';
-import { GET_NEEDS } from '@/services/queries';
```

Comment:
Also, add fulfilledBy to PropTypes :)

Reply:
Yeah, the fulfilledBy field is the reason I added it to the file instead. What happens when you add it to the shared query and then create a need is that it will throw a warning about the fulfilledBy field missing. It doesn't break though. Would you prefer that I look into why the warning occurs or is it fine if the component file has the query there?

Reply:
I suspect the only reason this works at all is that the query defined in this file has the same name as the shared query ("Needs"), which actually doesn't follow our convention for naming a query defined in a file (<ComponentName>_<returned field>). Someone might easily change this in the future and break it.

I'd say use the shared query from the file, and simply get rid of the warning by adding the fulfilledBy field to the mutation query in CreateNeed. (Then do the equivalent for ResponsibilitiesContainer and CreateResponsibility). That feels the "least wrong" and most robust to me. What do you think?

Reply:
Yup, you're right. Implemented your suggestion.